keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -294,7 +320,9 @@ the health of the service?**
     - [Optional] Aggregation method:
     - Components exposing the metric:
   - [ ] Other (treat as last resort)
-    - Details:
+    - Details: Check whether the workload moved to a different running node
```

Comment:
What metrics are available on pod GC controller that you're using to delete these pods?  I would expect to be to know if there's a problem removing these pods.  I would expect similar feedback on the attach/detach controller being modified.

Reply:
There isn't any metric in GC controller currently. We can add deleting_pods_total, deleting_pods_error_total.
For Attach Detach Controller, there's already a metric: attachdetach_controller_forced_detaches.